lib/imapoptions

```diff
@@ -654,6 +654,12 @@ Blank lines and lines beginning with ``#'' are ignored.
    if the CalDAV client hasn't specified a restriction in the creation
    request. */
 
+{ "calendar_minimum_alarm_interval", "5m", DURATION, "UNRELEASED" }
+/* The minimum allowed interval between alarms for a recurring event.
+   Primarily used to suppress alarms for MINUTELY and SECONDLY recurrences.
+   The default is 5 minutes.  The minimum value is 0, which will
+   allow all alarms, regardless of frequency. */
```

Comment:
`, regardless of frequency` seems unnecessary here.

Reply:
Removed